libs/common/src/lib/dto/user.dto.ts

```diff
@@ -47,4 +47,7 @@ export class UserDto extends JsonApiAttributes<UserDto> {
 
   @ApiProperty({ type: () => UserFramework, isArray: true })
   frameworks: UserFramework[];
+
+  @ApiProperty({ nullable: true })
+  id: number | null;
```

Comment:
Why is the user id needed by the client?

Reply:
Removed, rethinking it, it wont be necesary as the authenticatedId.